### PR TITLE
Update auth.php

### PR DESCRIPTION
**Pull request by itsho**
added debug prints to better figure out when LDAP has search issues.

### CODE REVIEW

File: lib/plugins/authldap/auth.php [hunk 1]
```diff
@@ -193,16 +197,33 @@ protected function _getUserData($user, $inbind = false) {
...
+			// no objects found
+			$this->_debug('LDAP search returned non-array result: '.htmlspecialchars(print($result)), -1, __LINE__, __FILE__);
+            return false; 
         }
-
+		
+		// Don't accept more or less than one response
+		if ($result['count'] != 1)		{
+			$this->_debug('LDAP search returned '.htmlspecialchars($result['count']).' results while it should return 1!', -1, __LINE__, __FILE__);
+			//for($i = 0; $i < $result["count"]; $i++) {
```
Comment (selfthinker) [inline lib/plugins/authldap/auth.php]: Having code in comments is not good practice. If it was supposed to not be there, better remove it. If it was supposed to be there for debugging purposes but only when actually debugging, remove the comment and add it only if `$this->getConf('debug')` or if `if($this->getConf('debug'))`.

Comment (itsho) [inline lib/plugins/authldap/auth.php]: I agree. do you want me to remove the code in comment? or you prefer to do it yourself?

Comment (splitbrain) [inline lib/plugins/authldap/auth.php]: please remove it in an additional commit.

Comment (itsho) [inline lib/plugins/authldap/auth.php]: I might be wrong but seems like someone else already did it

**Pull request merged**